pulpcore/cli/ansible/content.py

```diff
@@ -0,0 +1,29 @@
+import gettext
+
+import click
+
+from pulpcore.cli.ansible.context import PulpAnsibleCollectionVersionContext
+from pulpcore.cli.common.context import PulpContext, pass_pulp_context
+from pulpcore.cli.common.generic import list_command
+
+_ = gettext.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "content_type",
+    type=click.Choice(["collection-version"], case_sensitive=False),
+    default="collection-version",
```

Comment:
I'm wondering if we should expose this as a "collection" to the user.

Reply:
Let's go for consistency.

Reply:
the closest we get to the galaxy user experience, the better